pkgs/development/tools/phase/README.md

```diff
@@ -0,0 +1,31 @@
+# Phase CLI for nixpkgs
+
+This package provides the Phase CLI tool for secrets management.
+
+## Updating
+
+To update the package to a new version:
+
+1. Run the update script:
+
+This will pull latest integrity checksums and write them to `hashes.json` and bump the CLI version in `default.nix`.
+
+```bash
+cd pkgs/development/tools/phase
+./update.sh
+```
+
+2. Build and test the package:
+
+This will download the CLI build and install it locally for the suitable target architecture.
+
+```bash
+nix-build -A phase
+```
+
+3. Test basic functionality:
+
+```bash
+nix-shell
+phase --version
+```
```

Comment:
This really is not necessary. Also, the usual way to run an update script is by `nix-shell maintainers/scripts/update.nix --argstr package <attrpath>`